src/Modules/LearningTasks/Tutor.LearningTasks.Core/Mappers/ActivitiesProfile.cs

```diff
@@ -0,0 +1,20 @@
+﻿using AutoMapper;
+using Tutor.LearningTasks.API.Dtos.Activities;
+using Tutor.LearningTasks.Core.Domain.Activities;
+
+namespace Tutor.LearningTasks.Core.Mappers;
+
+public class ActivitiesProfile : Profile
+{
+    public ActivitiesProfile()
+    {
+        CreateMap<ActivityDto, Activity>();
```

Comment:
You can use:
```
CreateMap<ActivityDto, Activity>().ReverseMap();
```
as a simpler version of:
```
CreateMap<ActivityDto, Activity>();
CreateMap<Activity, ActivityDto>();
```

Reply:
Ok, I will replace this.